src/rastervision/tagging/tasks/validation_eval.py

```diff
@@ -18,13 +20,21 @@
 
 class Scores():
     """A set of scores for the performance of a model on a dataset."""
-    def __init__(self, y_true, y_pred, all_tags):
+    def __init__(self, y_true, y_pred, dataset):
         self.f2 = fbeta_score(y_true, y_pred, beta=2, average='samples')
+        self.f2_weighted = fbeta_score(y_true, y_pred, beta=2,
+                                       average='weighted')
+        self.atmos_scores, self.common_scores, self.rare_scores = {},{},{}
         f2_subscores = fbeta_score(y_true, y_pred, beta=2, average=None)
-        self.f2_subscores = dict(zip(all_tags, f2_subscores))
+        for tag in dataset.atmos_tags:
+            self.atmos_scores[tag] = f2_subscores[dataset.all_tags.index(tag)]
```

Comment:
This section of code is going to have to change after combining with the PR on active_tags, since the `all_tags.index` will no longer be a valid way to get indices. Not sure who will merge their PR first, but it would be a good idea to insert a TODO to update this.